src/landing/moudles/Route.tsx

```diff
@@ -0,0 +1,8 @@
+import { Switch, Route } from "react-router-dom";
+import Main from "./Main";
+
+const Routes = () => {
+  <Switch>
+    <Route path="/" exact component={Main} />
```

Comment:
Must: Are you sure this is pointing to Main component? I thought we would like to point to login component